electron/extraction.js

```diff
@@ -0,0 +1,39 @@
+// The code below is a copy of cli.js from mcode-extraction-framework
+
+const { MCODEClient } = require('mcode-extraction-framework/src/client/MCODEClient');
+const logger = require('mcode-extraction-framework/src/helpers/logger');
+const { mcodeApp } = require('mcode-extraction-framework/src/cli/app');
```

Comment:
I think you can use `require('mcode-extraction-framework')` for each of these, rather than also include the file path to the MCODEClient, logger, and app. The way we set up the exports from the MEF should let you import each directly. That would let you also combine this to one line, like this:

```
const { logger, mcodeApp, MCODEClient } = require('mcode-extraction-framework');
```